src/main/java/org/jabref/logic/util/strings/EmacsStringManipulator.java

```diff
@@ -0,0 +1,178 @@
+package org.jabref.logic.util.strings;
+import org.jabref.model.util.ResultingEmacsState;
+
+public class EmacsStringManipulator {
+    private enum LetterCase {
```

Comment:
I think, this is obsolete and `org.jabref.logic.bst.BibtexCaseChanger.FORMAT_MODE` can be used. See below for a longer explanation.